R/flatten.R

```diff
@@ -24,36 +24,36 @@
 #' x %>% map_int(1L)
 #' @useDynLib purrr flatten_impl vflatten_impl
 flatten <- function(.x) {
-  .Call(flatten_impl, .x)
+  .Call(flatten_impl, .x, PACKAGE = "purrr")
```

Comment:
hmm no, his comments were about my function calls and they do use strings. My bad.